ee/codegen/src/types/vellum.ts

```diff
@@ -897,3 +897,9 @@ export interface InlineWorkflowFunctionArgs {
   type: "INLINE_WORKFLOW";
   exec_config: WorkflowVersionExecConfig;
 }
+
+export interface DeploymentWorkflowFunctionArgs {
+  type: "DEPLOYMENT_WORKFLOW";
```

Comment:
[nit] agh, we actually usually call this `"WORKFLOW_DEPLOYMENT"` (yes the inverse of `"INLINE_WORKFLOW"`) in other contexts. Is this too late to change? If greater than a 5 min hassle, feel free to ignore